tests/test_packaging.py

```diff
@@ -1,2 +1,55 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function
+import random
+import os
+
+from testtools import TestCase
+
+from auxlib.packaging import _get_version_from_pkg_info, _is_git_dirty, _get_most_recent_git_tag, \
+    _get_git_hash, is_git_repo
+
+
+class TestPackaging(TestCase):
+
+    def test_version_string(self):
+        try:
+            test_version = str(random.randint(0,1e6))
+            with open('.version', 'w') as f:
+                f.write(test_version)
+            assert _get_version_from_pkg_info('tests') == test_version
+        finally:
+            if os.path.exists('.version'):
+                os.remove('.version')
+
+    def test_is_git_dirty(self):
+        result = _is_git_dirty()
+        assert result is True or result is False
+
+
+    def test_get_git_hash(self):
+        hash = _get_git_hash()
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Redefining built-in 'hash'](https://www.codacy.com/app/kalefranz/auxlib/file/2131696641/issues/source?bid=3138661&fileBranchId=3139130#l30)